sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidOperatorTable.java

```diff
@@ -437,6 +442,12 @@ public DruidOperatorTable(
 
     for (SqlOperatorConversion operatorConversion : operatorConversions) {
       final OperatorKey operatorKey = OperatorKey.of(operatorConversion.calciteOperator());
+      if (operationConversionDenySet.contains(operatorKey.name)) {
+        LOG.info(
+            "operatorKey '%s' not being added to available operatorConversions as it was found in deny list.",
+            operatorKey.name);
```

Comment:
Instead of relying on Calcite validation, should we throw an exception with a user-friendly message that would guide the user not to use blocked operators without exposing the internals?

Reply:
This could lead to information leak, as we may not want to expose which functions were explicitly put on deny list, but rather have it fail in generic way. Let's revisit this at later time, if necessary.

Reply:
This is a log message that would not normally be visible to the user. This is not a message given to the end user. The end user will get Calcite's usual message that "function foo with arguments VARCHAR, BIGINT, is not found."